src/scss/cdb-components/forms/radio.scss

```diff
@@ -16,57 +25,72 @@
 @import '../../cdb-variables/colors';
 
 .CDB-Radio {
-  position: relative;
   width: $baseSize * 2;
   height: $baseSize * 2;
-  border: 1px solid $cMainLine;
-  border-radius: 50%;
+  opacity: 0;
+  position: absolute;
+  z-index: 1;
 
-  &::before {
-    display: block;
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    width: $baseSize - 2;
-    height: $baseSize - 2;
-    margin-top: -3px;
-    margin-left: -3px;
+  ~ .CDB-RadioFace {
+    position: relative;
+    width: $baseSize * 2;
+    height: $baseSize * 2;
+    border: 1px solid $cMainLine;
     border-radius: 50%;
-    background: $cSecondaryLine;
-    content: '';
+    box-sizing: border-box;
+    cursor: pointer;
```

Comment:
Properties should be ordered position, width, height, border, border-radius, background, text-indent, cursor